### DISCUSSION TITLE
Chat stops responding, restart needed to resume

### DISCUSSION THREAD

**Discussion by tonino-manzi** [Genuitec/Copilot4Eclipse-community#18] [Help]
On this version

Version: 2024-06 (4.32.0)
Build id: 20240606-1231

![image](https://github.com/user-attachments/assets/cae47bf9-b8a3-49a9-bbd7-3a80d76a8fef)

Most of time Copilot not answer me. 
After a question the answer is empty

![image](https://github.com/user-attachments/assets/bd379c8b-8a58-4542-92e2-eac6d3ee7e0b)

When I restart Eclipse start to work again but after one or two answer I need to restart again.

In the error Log tab there isn't any error or message during the missing answer

Regards

**Comment by wayneparrott** [comment]
Thanks for sharing this issue.

Can you enable the low level logging described here: https://www.genuitec.com/products/copilot4eclipse/docs/basics/logging#logging-communications-with-github-copilot-services-low-level

This may provide some additional insight when Chat is not responding as expected. Please share the log details either here or if you want to keep such log details confidential send to support@genuitec.com to my attention (Wayne) and include a link to this thread.

**Reply by tonino-manzi** [reply]
I send you an email

**Comment by wayneparrott** [comment]
Thanks I followed up with an observation and suggestion for resolution.
W

**Reply by tonino-manzi** [reply]
I have done what you suggest. Now it works better but not always. I send you new logs
